aleph-client/src/lib.rs

```diff
@@ -1,19 +1,21 @@
-use codec::Encode;
 use std::{thread::sleep, time::Duration};
 
+use codec::Encode;
 use log::{info, warn};
 use sp_core::{sr25519, Pair, H256};
 use sp_runtime::{generic::Header as GenericHeader, traits::BlakeTwo256};
-use substrate_api_client::{
-    rpc::ws_client::WsRpcClient, std::error::Error, Api, ApiResult, RpcClient,
+pub use substrate_api_client::{
```

Comment:
well I found myself often including `sac`, (apart from `aleph-client`) in dependencies because I was missing one or two types
and yes, actually that would make more sense  to `pub use substrate_api_client`; can I? :P

Reply:
exports reworked